src/Form/DropdownField.jsx

```diff
@@ -15,12 +15,19 @@ export const DropdownField = ({ options, ...props }) => {
     [field, options]
   )
 
-  const selectedLabel = selectedOption ? selectedOption.label : undefined
+  const selectedLabel =
+    selectedOption && selectedOption.disabled !== true
+      ? selectedOption.label
+      : undefined
 
   return (
-    <Dropdown onChange={handleChange} label={props.label} value={selectedLabel}>
-      {options.map(({ label, value }) => (
-        <Dropdown.Option key={value} value={value}>
+    <Dropdown
+      onChange={handleChange}
+      label={props.label}
+      value={selectedLabel}
+      {...props}>
```

Comment:
What prop specifically do you need to pass down to `Dropdown`? You should add that prop not everything as this blurs the component's interface and makes it harder to update and maintain.

Reply:
All the layout props (margins, position) and all the HTML attributes valid for a selector (including `title`, `aria-`, etc).

When PropTypes are used properly, the maintainability of such a component is a breeze.